test/650_proxy_env_test.sh

```diff
@@ -31,7 +31,7 @@ check
 # Check we can use weave env/config with unix -Hs specified
 weave_on $HOST1 stop
 weave_on $HOST1 launch-proxy -H unix:///var/run/weave/weave.sock
-assert_raises "run_on $HOST1 'eval \$(weave env) ; docker $CMD'"
-assert_raises "run_on $HOST1 'docker \$(weave config) $CMD'"
```

Comment:
Since `launch-proxy` was called on the line above with `$DOCKER_HOST` set to `tcp:xxx`, `weave.sock` doesn't get the docker group ownership.